corehq/apps/app_manager/management/commands/helpers.py

```diff
@@ -79,6 +114,18 @@ def log_info(self):
     def log_debug(self):
         return self.options.get("verbosity", 0) > 2
 
+    @property
+    def use_chunks(self):
+        return self.options.get('use_chunks', False)
+
+    @property
+    def total_chunks(self):
+        return int(self.options.get('total_chunks', None))
```

Comment:
`int(None)` will fail - should this default to `1`?  also, you can do the fallback and type coercion in `parser.add_argument` like this random example:
https://github.com/dimagi/commcare-hq/blob/fd574f67ddb3061685de878cd33c02b59c8dbf69/corehq/form_processor/management/commands/run_sql.py#L29-L30
That way argparse will provide feedback if the user passes in a bad value.

Also, FWIW argparse makes all options always in the `options` dict, so you can use `options['total_chunks']` safely, and it'll be `None` or the specified default if the user doesn't pass that arg in.